styles.css

```diff
@@ -0,0 +1,476 @@
+}
+
+.summit-section img {
+  width: 97%;
+}
+
+#program {
+  background-image: url(./elements/prgrmssctn1.jpeg);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100%;
+  padding: 5%;
+}
+
+.program-intro {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 5%;
+  color: var(--grey);
+}
+
+.program-card__title {
+  color: var(--orange);
+  font-weight: bold;
+}
+
+.program-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 10px;
+  padding: 5% 5%;
+}
+
+.program__card {
+  border: 1px solid var(--grey);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background-color: var(--black);
+  color: var(--grey);
+  gap: 13px;
+  padding: 15px;
+  width: auto;
+}
+
+.card-image {
+  width: 50%;
+  height: 12vw;
+  object-fit: cover;
+}
+
+.program-link {
+  display: none;
+}
+
+.program__button {
+  padding: 5%;
+  color: var(--grey);
+  background-color: var(--orange);
+  border: none;
+  margin-bottom: 10%;
+}
+
+.speaker-container {
+  padding: 3rem 0.1rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 1rem;
+}
+
+.speaker-card {
+  max-width: 95%;
+  background-color: white;
+  padding: 0.5rem;
+  border-radius: 10px;
+  align-items: flex-start;
+  gap: 10px;
+  display: flex;
+  flex-direction: row;
+  border: 1px solid var(--grey);
+}
+
+.speaker-card img {
+  border-radius: 5px;
+}
+
+.speaker-card__text {
+  display: flex;
+  flex-direction: column;
+}
+
+.speaker-card__title {
+  color: var(--black);
+  font-size: 1em;
+  margin: 1rem 0;
+}
+
+.speaker-card__subtitle {
+  color: var(--orange);
+  font-size: 0.9em;
+  font-style: oblique;
+}
+
+#footer {
+  display: none;
+}
+
+.footer-partners {
+  display: flex;
+  flex-direction: column;
+  padding: 5%;
+  background-color: var(--black);
+}
+
+.footer-partners__intro {
+  display: flex;
+  flex-direction: column;
+  color: var(--grey);
+  align-items: center;
+}
+
+.footer-partners__logo {
+  display: flex;
+  justify-content: space-evenly;
+  padding: 18px 1px;
+}
+
+.partner-logos {
+  padding: 0% 3%;
+  width: 13%;
+}
+
+.footer-logo {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 8px;
+}
+
+.footer-copyright {
+  display: flex;
+  padding: 5% 4%;
+  gap: 10px;
+}
+
+.footer-copyright img {
+  width: 30%;
+}
+
+.footer-title {
+  font-weight: bold;
+  color: var(--black);
+}
+
+.footer-detail__text {
+  color: darkgrey;
+}
+
+.speakers-bttn {
+  width: 80%;
+  margin-bottom: 20px;
+  margin-left: 35px;
+  padding: 9px;
+  background-color: #fff;
+  border-radius: 5px;
+  font-size: larger;
+}
+
+@media (min-width: 768px) {
+  .topPageDiv {
+    display: flex;
+    background-color: var(--black);
+    color: var(--grey);
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .navbar {
+    display: flex;
+
+    --bs-navbar-hover-color: #ec5342c8;
+  }
+
+  .bg-light {
+    --bs-bg-opacity: 0.2;
+  }
```

Comment:
- [ ] While you did a good job styling your Navbar :+1:, as the project requires the page should look almost identical to the original design only small adjustments like text or image changes are acceptable. So you should use a white background instead a transparent one just like the template :+1: 
- [ ] Also, kindly check that the page links are aligned to the right just like the template :+1: 

**Your design**

![Screenshot from 2023-02-09 06-08-45](https://user-images.githubusercontent.com/105117832/217797454-212647af-8ff9-4bdf-ab1f-3030c5a5c06b.png)

![Screenshot from 2023-02-09 06-09-02](https://user-images.githubusercontent.com/105117832/217797487-d5b546ac-add7-4ee3-9f4a-e564c1c88952.png)

**Template**

![Screenshot from 2023-02-09 06-14-53](https://user-images.githubusercontent.com/105117832/217797709-28840883-1903-4a52-b3ee-d8cbd68a1db5.png)